alembic/versions/2024_01_24_a6befebf1231_add_order_table.py

```diff
@@ -0,0 +1,48 @@
+"""Add order table
+
+Revision ID: a6befebf1231
+Revises: 584840c706a0
+Create Date: 2024-01-24 10:43:55.206551
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a6befebf1231"
+down_revision = "584840c706a0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "order",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("customer_id", sa.Integer(), nullable=True),
+        sa.Column("order_date", sa.DateTime(), nullable=True),
```

Comment:
also here, why can it be nullable, if an order is placed it should have a date

Reply:
Yeah, that is also true.

Reply:
Changed it.